apps/example-orchestrator-ui/components/StartPage/NewProcessPanel.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement, useState } from 'react';
+import {
+    EuiButton,
+    EuiFieldSearch,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiPanel,
+    EuiSpacer,
+    EuiText,
+    EuiTextColor,
+} from '@elastic/eui';
+
+export default function NewProcessPanel(): ReactElement {
+    const [value, setValue] = useState('');
+    const onChange = (e) => {
+        setValue(e.target.value);
+    };
+    const frequentlyUsedValues = [
+        'IP Prefix',
+        'SN8 LightPath',
+        'NL8 MSC',
+        'NL8 Service Port 100G',
+        'SN8 AGGSP',
+    ];
+    const renderFrequentlyUsed = (values) => {
```

Comment:
This could be a component, the usage on line 56 will look nicer <FrequentlyUsed values=frequentlyUsedValues />